### ISSUE TITLE
How can I extend the rules?

### ISSUE THREAD

**Issue by contributorpw** [sindresorhus/awesome-lint#144]
I need to replace the string 
https://github.com/sindresorhus/awesome-lint/blob/main/rules/list-item.js#L57

```js
toString(node) === 'Contents'
```

to something like this

```js
customRules.nodeEquals('contents', toString(node))
```

because my node equals `Содержание`. I maintain the list in another language.

So my question. How can I extend the rules? Should we expect some configuration file or something like that?

**Comment by sindresorhus** [comment]
You cannot really extend rules at that level, but there could be a rule option for what text to match against.

**Comment by contributorpw** [comment]
OK. At now I don't see an effective rule. I need to learn the code for a good idea and PR.
